ckanext/openbudgetsin_theme/templates/base.html

```diff
@@ -7,3 +7,26 @@
 {% resource 'openbudgetsin_theme/openbudgetsin_theme_base.css' %}
 {% endblock %}
 
+{% block meta %}
+      {{ super() }}
+  <meta charset="utf-8">
+    <meta name="description" content="Open Budget Data Portal for India">
+    <meta name="application-name" content="openbudgetsindia.org">
+    
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:site" content="@OpenBudgetsIn">
+    <meta name="twitter:title" content="OpenBudgetsIndia
+      ">
+    <meta name="twitter:description" content="Open Budget Data Portal for India">
+    <meta name="twitter:url" content="https://openbudgetsindia.org/">
+    <meta name="twitter:image:src" content="">
+    
+    <meta property="og:title" content="OpenBudgetsIndia" />
+    <meta property="og:description" content="Open Budget Data Portal for India" />
+    <meta property="og:type" content="article" >
+    <meta property="og:url" content="https://openbudgetsindia.org/" >
+    <meta property="og:image" content="Logo_highres_text_original-26.png" >
```

Comment:
@akshay2905: We need to get a banner with light blue background from @SuchismitaNaik which shall also have byline: 'Making India's Budgets open, usable and easy to comprehend'